### PR TITLE
Module 1: First project "Hello Microverse"

### PR DESCRIPTION
**Pull request by albertkantwi**
In this project I have edited and commit index.html, style.css and all files. I humbly request for your review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,244 @@
...
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+
+
+**[Hello Microverse]** is a projecct to help work with tools like linters and others.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
```
Comment (BregornOriginal) [inline README.md]: Good work implementing the template provided by Microverse! 💯 

- But there are some changes you need to do to it, kindly change the information on it like the tech stack with the technologies that you used in this project, in this case, for example, can be `html` `css`

File: README.md [hunk 2]
```diff
@@ -1 +1,244 @@
...
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
```
Comment (BregornOriginal) [inline README.md]: - Kindly delete or replace this technology as you are not using this one on this particular project

File: README.md [hunk 3]
```diff
@@ -1 +1,244 @@
...
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
```
Comment (BregornOriginal) [inline README.md]: - Kindly delete or replace this technology as you are not using this one on this particular project

File: README.md [hunk 4]
```diff
@@ -1 +1,244 @@
...
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
```
Comment (BregornOriginal) [inline README.md]: - Kindly delete or replace this technology as you are not using this one on this particular project

File: README.md [hunk 5]
```diff
@@ -1 +1,244 @@
...
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
```
Comment (BregornOriginal) [inline README.md]: - Kindly delete the section or add some features that you think would fit in here.

File: README.md [hunk 6]
```diff
@@ -1 +1,244 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://google.com)
```
Comment (BregornOriginal) [inline README.md]: - As your project doesn't have a live demo already, kindly mention that is going to be available soon or delete this section

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @albertkantwi  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

giphy (3)


## Highlights

✅ Exellent branch name
✅ Good Pull Request description 💯 

## Required Changes ♻️

- Kindly setup your linters to make your project looks cleaner and save you time while writing code, you can find how to do it [here](https://github.com/microverseinc/linters-config)
- Commits messages should be meaningful, so instead of writing a commit like `adding` or `edited`, it should be `add` or `edit`

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @albertkantwi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**